discord-actions/README.md

```diff
@@ -0,0 +1,141 @@
+# discord-actions
+
+## **Requests**
+
+|                          Route                          |           Description            |
+| :-----------------------------------------------------: | :------------------------------: |
+|  [GET /discord-actions/invites](#get-discord-actions-invites)   | return all the invites generated |
+| [POST /discord-actions/invites](#post-discord-actions-invites) |  generates a invite for a user   |
+
+---
+
+## **GET /discord-actions/invite**
+
+- return the discord invite created for the user
+
+- **Query**
+
+  - Optional: `userId=[string]`: if this is provided it returns the invite of the user whose userId is provided in the query param, if the user is not a super user then this should be equal to the userId of the user, if no userId is provided, it return invite for the user who has made the request
```

Comment:
Whats the use of this? I'm not getting it

Reply:
This is done in case the super user wants to see the link generated for a particular user